lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -16,9 +17,9 @@
     protected $_name;
 
     /**
-     * @var string the text for this suggestion
+     * @var string the prefix for this suggestion
      */
-    protected $_text;
+    protected $_prefix;
```

Comment:
I think both still exists. I'm not a big fan of local variables as we have `params` which brings more flexilibty. I was thinking if we should remove the variable completely and use params for prefix and text.